### DISCUSSION TITLE
Improve system email validation

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#10738] [Feature Requests]
There are at least 3 places where a user's email is concerned
- installation
- login
- user's detail

The login requires a valid email, but the installation and the user's detail don't.

As a result, in case of a mistake when editing the address, it is not possible to login.
Blocking the whole installation in case of the admin.

Even if this will not prevent from entering a wrong email address, it would be nice to have a control on a **valid format** address when installing or modifying a user's address.
Another solution would be to consider the login address as a mere username and get rid of the valid email requirement. Not to say that the email recover will then not be available.

### To Reproduce

On
- install, answer "myemail" (without @mydomain.com)
or on
- user's detail: modify the email to "myemail" (without @mydomain.com)

### Errors Shown

The user cannot login anymore.
And if is the admin (sure admin should pay attention ;), there is no way to recover by modifying the row (as it is fortunately protected as read-only in the database).

**Comment by rijkvanzanten** [comment]
Once we have https://github.com/directus/directus/discussions/4390 and https://github.com/directus/directus/discussions/7476 implemented, this should be fairly easy to resolve.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
